### DISCUSSION TITLE
I don’t understand disk size exceeding

### DISCUSSION THREAD

**Discussion by glauber-sampaio** [supabase/supabase#29847] [Questions]
Hi! First, I am not a developer. I develop stuff for myself, play around with a few things, but definitely not developer enough to understand database issues.

That being said, can someone help me with understanding why the disk size is exceeding? I read the docs but I still can’t understand what disk size means.

This database (random) I use for local stuff, testing small projects locally, so I don’t understand why it’s exceeding every day. If I remove the spend cap I’ll have a considerable bill to pay – for nothing.

![image](https://github.com/user-attachments/assets/c76ad7b4-a83d-467c-8152-1e5b75aaca19)

![image](https://github.com/user-attachments/assets/2dca5723-48a0-44cd-82d5-549b6c788b12)

![image](https://github.com/user-attachments/assets/c73ef617-d094-441c-90ce-60e2b51bfc7a)

**Comment by kantarse** [comment]
I have exactly the same question. It's disappointing to see that nobody answered this. 
My disk got automatically upsized from 8 to 12 GB. Although the chart shows only less than 4 GB is used. Furthermore, when I want to downsize it back to 8 GB, it doesn't allow me to.
Now I will have to open a support ticket...
